src/porepy/numerics/time_step_control.py

```diff
@@ -745,7 +745,7 @@ def write_time_information(self, path: Optional[Path] = None) -> None:
 
         # Storing as json
         if path is None:
-            path = Path("visualization") / Path("times.json")
+            path = Path("visualization") / "times.json"
```

Comment:
My sense is, currently, there is two defaults provided, which is unfortunate. My feeling would be to remove the default use here. Then one possibility would be to say, that if no path is provided, no times.json file should be exported (as anyhow already implemented below).

Reply:
I see your point. Unfortunately, `TimeManager` is not able to access `model.params`, hence `DataSavingMixin.write_pvd_and_vtu` has to pass a path and we cannot just have one default in `write_time_information`. Your suggestion sounds sensible. I will also remove the default path in `load_time_information` then.

Reply:
Pushed the update :)